lib/bitbucket_rest_api/client.rb

```diff
@@ -14,12 +14,6 @@ def issues(options = {})
       @issues ||= ApiFactory.new 'Issues', options
     end
 
-    # An API for users to manage their own tokens.
-    def oauth(options = {})
-      @oauth ||= ApiFactory.new 'Request::OAuth', options
-    end
-    alias :authorizations :oauth
-
```

Comment:
This appears to be an interface into the old OAuth1 implementation so I removed it.